pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -58,15 +58,17 @@ import (
 // controllerKind contains the schema.GroupVersionKind for this controller type.
 var controllerKind = batchv1beta1.SchemeGroupVersion.WithKind("CronJob")
 
-type CronJobController struct {
+// Controller is a controller to manage cronjobs
```

Comment:
I like the removal of comment redundancy above. Can we make this ?
// Controller to manage cronjobs